src/app/services/admin-patient.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@angular/core';
+import { ServerApiService } from './server-api.service';
+import { Observable } from 'rxjs';
+
+export class GenericAdminPatientResponse {
+  public message: string;
+}
+
+export class ResetPasswordResponse {
+  public password: string;
```

Comment:
ResetPasswordResponse should be declared in external object under the `api-objects` folder